makefiles/tools/openocd.inc.mk

```diff
@@ -16,10 +16,17 @@ endif
 
 OPENOCD_CONFIG ?= $(BOARDDIR)/dist/openocd.cfg
 
-# Export OPENOCD_CONFIG to required targets
 OPENOCD_TARGETS = debug% flash% reset
+
+# Export GDB_PORT to required targets
+$(call target-export-variables,$(OPENOCD_TARGETS),GDB_PORT_CORE_OFFSET)
```

Comment:
Comment does not match what is exported.

Reply:
done